cassandra/policies.py

```diff
@@ -1286,11 +1293,13 @@ def encrypt(self, coldesc, obj_bytes):
 
         cipher = self._get_cipher(coldesc)
         encryptor = cipher.encryptor()
-        return encryptor.update(padded_bytes) + encryptor.finalize()
+        return self.iv + encryptor.update(padded_bytes) + encryptor.finalize()
 
-    def decrypt(self, coldesc, encrypted_bytes):
+    def decrypt(self, coldesc, bytes):
 
-        cipher = self._get_cipher(coldesc)
+        iv = bytes[:AES256_BLOCK_SIZE_BYTES]
+        encrypted_bytes = bytes[AES256_BLOCK_SIZE_BYTES:]
+        cipher = self._get_cipher(coldesc, iv=iv)
```

Comment:
Worth being explicit about this: the IV supplied via the policy constructor is never used for decrypted read values.  We _only_ use that when writing new data.  Seems obvious given how the code is organized but may not be obvious to users... perhaps we should more explicitly call this out?

Reply:
I think it should be fine.